twingate/internal/client/client_test.go

```diff
@@ -88,7 +88,7 @@ func TestClientAPITokenNotSet(t *testing.T) {
 	os.Setenv(EnvAPIToken, "xxx")
 	_, err = client.post(context.TODO(), "/hello", "hello", nil)
 
-	assert.ErrorContains(t, err, "lookup test.twindev.com on 127.0.0.53:53: no such host")
+	assert.ErrorContains(t, err, "lookup test.twindev.com")
```

Comment:
@eran I was running tests locally on mac, and getting here different error message, so I guess it make sense to shorten it to the common part